README.md

```diff
@@ -1,58 +1,32 @@
 # erblint-github
+
 Template style checking for GitHub's Ruby projects
 
 ## Setup
 
-1. Update your `Gemfile` and run `bundle install`
+1. Update your `Gemfile` and run `bundle install`.
 
 ``` ruby
 gem "erb_lint", require: false
 gem "erblint-github"
 ```
 
-2. Require the linters within the `.erb-linters` folder. This could be done by adding a file `.erb-linters/erblint-github.rb` with the following line.
+2. **Important**: Create a new file with the path `.erb-linters/erblint-github.rb` and add the line:
```

Comment:
It'd be nice to eventually get rid of this step. I believe this requires updates to erb-lint.